### PR TITLE
fix(DocSite): Fix IE11 issues

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

The current [doc site](https://fluentsite.z22.web.core.windows.net/) it's completely not working on IE11 and this is because of [every grid child element need to be positioned by hand using -ms-grid-row and -ms-grid-column](https://stackoverflow.com/questions/52162463/css-grid-ie-11-overlap) to avoid overlap between rows. 

Screenshot 2020-06-05 at 17 08 35

This PR fixes that by replacing the Grid usage for Flex and setting styles to avoid big changes. 

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/fluentui/docs/src/components/ComponentPlayground/ComponentPlaygroundTemplate.tsx [hunk 1]
```diff
@@ -19,43 +19,45 @@ const NoopKnobProvider: React.FunctionComponent = props => {
 };
 
 const ComponentPlaygroundTemplate: React.FunctionComponent<ComponentPlaygroundTemplateProps> = props => (
-  <Grid
-    styles={{
-      gridTemplateColumns: '1fr 300px',
-      msGridColumns: '1fr 1rem 300px',
```
Comment (layershifter) [inline packages/fluentui/docs/src/components/ComponentPlayground/ComponentPlaygroundTemplate.tsx:25]: Ohoh, this breaks layout

image


vs 

image

File: packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx [hunk 2]
```diff
@@ -18,15 +18,9 @@ const ExampleSection: React.FC<ExampleSectionProps> = ({ title, children }) => (
     <Header as="h2" align="center" styles={headerStyle} className="no-anchor">
       {title}
     </Header>
-    <Grid
-      variables={{ gridGap: '2rem' }}
-      styles={{
-        gridTemplateColumns: 'minmax(550px, 1fr)',
```
Comment (layershifter) [inline packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx:24]: And this breaks also:

image

image

Comment (assuncaocharles) [reply packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx:24]: The problem seems to be an issue with `Toolbar` component when inside a container with `display: flex`. To temporary fix it I changed the example itself, but I guess it is a toolbar issue. Right @pompomon @layershifter @jurokapsiar ?

Comment (assuncaocharles) [reply packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx:24]: Fixed as agreed stretching the container for now, but I think we might still wanna to report the issue for the toolbar?

File: packages/fluentui/docs/src/components/ComponentPlayground/ComponentPlaygroundTemplate.tsx (hunk 1 above)
Comment (assuncaocharles) [reply packages/fluentui/docs/src/components/ComponentPlayground/ComponentPlaygroundTemplate.tsx:25]: Fixed in the best way that keeps the same for chrome and makes it usable for IE

File: packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx (hunk 2 above)
Comment (layershifter) [reply packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx:24]: > Fixed as agreed stretching the container for now, but I think we might still wanna to report the issue for the toolbar?

Yup, let's do it

**Pull request merged**